content/articles/letsencrypt.markdown

```diff
@@ -32,9 +32,25 @@ Let's Encrypt is an innovative certificate authority from several point of views
 - open: the source code of the Let's Encrypt certification authority is completely open source, and available in a [GitHub account](http://github.com/letsencrypt). This is by far the most unique characteristic of this CA.
 
 
+## Differences between Let's Encrypt and Standard SSL certificates
+
+The table below summarizes the most important differences between Let's Encrypt and Standard SSL certificates.
+
+|               | Let's Encrypt | Standard      |
+|---------------+---------------+---------------|
+| Certificate Expiration | 90 days | 1-3 years |
+| Single names | Supported | Supported |
+| Wildcard names | Not Supported | Supported |
+| Multi-domain (SAN) | Supported by default | Supported only by specific products |
```

Comment:
Should we mention plan support like you do on the `standard-vs-letsencrypt` page?